pyanaconda/storage_utils.py

```diff
@@ -519,6 +519,34 @@ def verify_mounted_partitions(storage, constraints, report_error, report_warning
                                "installation. Please unmount it and retry.") % part.path)
 
 
+def verify_lvm_block_sizes(storage, constraints, report_error, report_warning):
+    """ Verify block sizes of VGs on top of PVs.
+
+    LVM no longer allows creating VGs on top of PVs with different block sizes.
+
+    :param storage: a storage to check
+    :param constraints: a dictionary of constraints
+    :param report_error: a function for error reporting
+    :param report_warning: a function for warning reporting
+    """
+    for vg in storage.vgs:
+        if vg.exists:
+            continue
+
+        disks = set(pv.disk for pv in vg.pvs)
+        sector_sizes = set(d.format.sector_size for d in disks)
```

Comment:
Since its possible for the PVs to be unpartitioned disks you should check that the format type is 'disklabel' before accessing the `sector_size` attribute.

Reply:
I didn't realize this. Looks like we need to add a `sector_size` property for `DiskDevice`.